tools/check-vendored.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+_EXPECTED_VERSION_MIMEPARSE="1.6.0"
```

Comment:
In the interest of KISS, the version number is hard-coded here instead of going to the trouble up reading it from `mimeparse.py`. We can always change this later.